pyproject.toml

```diff
@@ -43,6 +43,7 @@ colorama = ">=0.4.6"
 pydantic = "^1.10.9"
 google-api-python-client = {version = ">=2.89.0", optional = true}
 chromadb = ">=0.3.26"
+lancedb = ">=0.1.16"
```

Comment:
can you move this to `[tool.poetry.extras]` section? i am planning to move chromadb and pinecone to there as well to make the package smaller

Reply:
@yyiilluu done! followed this https://python-poetry.org/docs/pyproject/#extras